jwst/lib/tests/helpers.py

```diff
@@ -12,6 +12,12 @@
 
 
 class MockS3Client:
+    def __init__(self, s3_test_data_path=None):
+        if s3_test_data_path:
+            self.s3_test_data_path = s3_test_data_path
+        else:
+            self.s3_test_data_path = S3_TEST_DATA_PATH
```

Comment:
If we want to discourage writing tests that rely on data in S3_TEST_DATA_PATH, maybe the client should just raise errors when the path in the initializer is None.  Something like "whoops, your test is reading from S3, but you didn't set that up".